ESS/v3/_summit.yaml

```diff
@@ -80,3 +80,20 @@ instances:
             ftdi_id: A104SE9D
             sensor_type: Temperature
             location: M1 sensor 1, M1 sensor 2, M1 sensor 3, M1 sensor 4, M1 sensor 5, M1 sensor 6
+      - client_class: LabJackDataClient
+        # LabJack reading AuxTel camera coolant pressures
+        config:
+          device_type: T4
+          connection_type: TCP
+          identifier: 139.229.170.151  # TODO IT-3689: use the IP name, once it is updated
```

Comment:
The DNS record for the IP name already is in place. However, there is an inconsistency in the name ` aux-adam01.cp.lsst.org` since the AuxTel raspberry pi is called `auxtel-ess01.cp.lsst.org`. Please clarify with IT what is the naming convention and update this line accordingly before merging into develop.